frontend/packages/console-app/src/components/modals/service-binding/BindableServices.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { FormGroup, Alert } from '@patternfly/react-core';
+import { useFormikContext, FormikValues } from 'formik';
+import * as fuzzy from 'fuzzysearch';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { getBindableServiceResources } from '@console/dev-console/src/components/topology/bindable-services/bindable-service-resources';
+import { FirehoseResource } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { ResourceDropdownField, getFieldId } from '@console/shared';
+
+export const bindableResources = (namespace: string) => {
+  const bindableRes = _.omit(getBindableServiceResources(namespace), 'serviceBindingRequests');
```

Comment:
Currently, the topology filters through bindable resources and allows binding with only the resources that have the label `app.kubernetes.io/component=external-service`. Should we do the same for the modal as well?

Reply:
@sahil143 @divyanshiGupta  ^

Reply:
If through topology connectors binding is only possible with the resources that have the above label then the modal dropdown should show only those resources.

Reply:
@rottencandy @divyanshiGupta I verified this on a cluster. I am able to drag and drop the handle and create a service binding with a service that does not have the `external-service` label. Also, I noticed a few things: The drop hints are different, for the Postgres Cluster the drop hint says `Create a binding connector` and for the Kafka conc it says `Create Service Binding`. Also, when I tried adding the `external service` label to the postgres cluster grouped  node, the dotted lines disappeared and the resource named `hippo` transformed into a `Trapezoid` shaped node
 
![sb](https://user-images.githubusercontent.com/22490998/144291748-5d9d3df0-23fe-4f9b-b441-b3292df70c33.gif)

Reply:
@sahil143 do you have an idea about this?

Reply:
I think it makes sense that once the `external-service` label is added that shape turned into a trapezoid because we show the resources that have the external-service label and are bindable(listed in bindable CRD) as Trapezoid shape. Currently, we allow binding connectors on OBS and the resources listed in the Bindable CRD.